packages/esm-service-queues-app/src/queue-table/queue-table-metrics.component.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
```

Comment:
I'm not sure this needs to be in it's own component, it can always be refactored out as needed in the future.  I'd probably just include this directly in the page where it is used for now